README.md

```diff
@@ -115,121 +114,83 @@ The plugin will write out all metrics serialized as JSON to the specified file.
 ```
 
 ### Examples
-See full task manifests using the mock and psutil plugins which are available in [examples/tasks](examples/tasks)  
 
+Example of running [psutil collector plugin](https://github.com/intelsdi-x/snap-plugin-collector-psutil), [movingaverage processor plugin](https://github.com/intelsdi-x/snap-plugin-processor-movingaverage), and writing data as a JSON to file:
 
-Example of running mock collector plugin, passthru processor plugin, and writing data as a JSON to file:
+Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 
-Make sure that your `$SNAP_PATH` is set, if not:
-```
-$ export SNAP_PATH=<snapDirectoryPath>/build
-```
-In one terminal window, open the snap daemon (in this case with logging set to 1 and trust disabled):
-```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0
-```
-In another terminal window:  
-Load snap-plugin-publisher-file plugin:
-```
-$ $SNAP_PATH/bin/snapctl plugin load build/rootfs/snap-plugin-publisher-file
-
-Plugin loaded
-Name: file
-Version: 2
-Type: publisher
-Signed: false
-Loaded Time: Mon, 25 Jul 2016 12:30:25 CEST
-```
-Load snap-plugin-processor-passthru plugin:
-```
-$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-plugin-processor-passthru
-
-Plugin loaded
-Name: passthru
-Version: 1
-Type: processor
-Signed: false
-Loaded Time: Mon, 25 Jul 2016 12:33:00 CEST
-```
+Ensure [snap daemon is running](https://github.com/intelsdi-x/snap#running-snap):
+* initd: `service snap-telemetry start`
+* systemd: `sysctl start snap-telemetry`
+* command line: `snapd -l 1 -t 0 &`
 
-Load snap-plugin-collector-mock2 plugin:
-```
-$ $SNAP_PATH/bin/snapctl plugin load $SNAP_PATH/plugin/snap-plugin-collector-mock2
-
-Plugin loaded
-Name: mock
-Version: 2
-Type: collector
-Signed: false
-Loaded Time: Mon, 25 Jul 2016 12:32:09 CEST
-```
 
-See available metrics for your system:
+Download and load Snap plugins:
 ```
-$ $SNAP_PATH/bin/snapctl metric list
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-publisher-file/latest/linux/x86_64/snap-plugin-publisher-file
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-processor-movingaverage/latest/linux/x86_64/snap-plugin-processor-movingaverage
+$ wget http://snap.ci.snap-telemetry.io/plugins/snap-plugin-collector-psutil/latest/linux/x86_64/snap-plugin-collector-psutil
+$ chmod 755 snap-plugin-*
```

Comment:
This is not required, snapd saves uploaded plugins in temp and sets chmod to 700.